app/models/lockstep/status.rb

```diff
@@ -0,0 +1,13 @@
+class Lockstep::Status
+  Lockstep::ApiRecord.model_name_uri = 'v1/Status'
+
```

Comment:
where is the id_ref for this ? I assume all model does need the id_ref params

Reply:
is this endpoint present in swagger ?  in that case , i assume even load_schema is needed na ?

Reply:
id_ref is not needed for this as this is not a record from the platform :) Even load_schema isn't necessary for this. This endpoint is present in schema but it's just the current user status that will be returned, which is not a record in the database but a set of attributes relating to user.